docs/pages/get-started/faq.md

```diff
@@ -410,83 +416,141 @@ Currently, messaging incurs no fee. As XMTP decentralizes, messaging between par
 
-XMTP may occasionally delete messages and keys from the `dev` network and will provide advance notice in the [XMTP Discord community](https://discord.gg/xmtp) and [XMTP Announcements forum](https://community.xmtp.org/c/start-here/announcements/7).
+For example, a retention policy may look something like the following, though specifics are subject to change:
+  - One year for messages
+  - Indefinite storage for account information and personal preferences  
 
-Different approaches to long-term message storage are currently being researched.
+The team is researching a way to provide this indefinite storage and have it scale forever. 
+- If research shows that it's possible, we'll share a plan for how it will be archieved. 
+- If research shows that it isn't possible, we'll share a plan that shows how retention periods will provide a permanent solution to storage scaling.
+
+Have questions or feedback regarding message storage and retention? Post to the [XMTP Community Forums](https://community.xmtp.org/c/development/ideas/54).
+
+#### For the on-device database managed by the XMTP SDK:
+
+Messages are stored for as long as the user decides to keep them. However, encryption keys are regularly rotated.
+
+### What are the costs of XMTP message storage and retrieval?
+
+Messages are stored off-chain on the XMTP network, with all nodes currently hosted by Ephemera. Ephemera currently absorbs all message storage and retrieval costs.
+
+There are no message storage and retrieval-related fees incurred by developers for building with the XMTP SDK.
 
 ## Messages
 
-### Which message formats and metadata does XMTP support?
+### Which message formats does XMTP support?
 
-XMTP transports a message payload as a set of bytes that can represent any format that a developer wants to support, such as plain text, JSON, or even non-text binary or media content.
+XMTP transports a message payload as a set of bytes that can represent any format that a developer wants to support, such as plain text, JSON, or non-text binary or media content.
 
 With XMTP, these message formats are called content types. Currently, there are two basic content types available. These content types aim to establish broad compatibility among apps built with XMTP.
 
 The XMTP community can propose and adopt standards for other content types, either informally or through a governance process.
 
-Message payloads also include references to timestamps. However, timestamps are not currently independently verified and can be set to any value by the sending app.
-
 To learn more about content types, see [Content types](/content-types/content-types).
 
 To learn more about the XMTP improvement proposals governance process, see [What is an XIP?](https://github.com/xmtp/XIPs/blob/main/XIPs/xip-0-purpose-process.md)
 
-Have other questions or ideas about message formats and metadata? Post to the [XMTP Community Forums](https://community.xmtp.org/).
+### Which message metadata does XMTP support?
```

Comment:
Correct in v2 and v3?

Reply:
I'm unsure about the intent of this question - is it intended to be about what extra metadata apps are able to attach to message payloads, or is it about what metadata gets leaked onto the network?

Reply:
Thank you for this question, @richardhuaaa - it is the latter - are there any changes to the types of metadata leaked onto the network? =)

Reply:
This one is a bit complex and requires a lot of precision - I wonder if it would be best to link to a section in our [audit docs](https://hackmd.io/@jan-cryspen/rJ5W-83j0#Privacy) for now, @neekolas. The [MLS spec](https://www.rfc-editor.org/rfc/rfc9420.html#section-16.4) also has some general info on MLS but it may not be enough for our specific implementation.